### DISCUSSION TITLE
where are the available icons?

### DISCUSSION THREAD

**Discussion by JuanVqz** [pop-os/launcher#16] [Q&A] [answered]
I did my own simple script to start [tmux](http://www.sromero.org/wiki/linux/aplicaciones/tmux)

where are the available icons and how can I add my own?

https://github.com/pop-os/launcher/blob/4a5f9b09c0129a43068274232d9beec871009073/scripts/session/session-logout.sh#L4

btw, this is my script.
```bash
#  /home/username/.local/share/pop-launcher/scripts/tmux/start-tmux.sh
#!/bin/sh
#
# name: Start Tmux
# icon: system-restart
# description: Start my tmux sessions
# keywords: start my day

alacritty --command tmux
```

I was about to create an issue ☺️ when I saw the discussions tab.

**Comment by mmstick** [comment, marked as answer]
The icon may be a path or a name, with the name referring to an icon in `/usr/share/icons`, preferring the system icon theme.

There's a Flatpak application called Icon Library, which lists available symbolic icons.

**Reply by JuanVqz** [reply]
Hey mmstick, thank you for your answer!
